client/package.json

```
@@ -239,9 +239,10 @@
 		"autoprefixer": "^10.2.5",
 		"component-event": "^0.2.0",
 		"component-query": "^0.0.3",
-		"react-test-renderer": "^16.12.0",
+		"eslint-plugin-you-dont-need-lodash-underscore": "^6.12.0",
 		"pkg-dir": "^5.0.0",
 		"postcss-custom-properties": "^11.0.0",
+		"react-test-renderer": "^16.12.0",
```

This is probably just a hangover from some earlier work you were doing?

```suggestion
		"react-test-renderer": "^16.12.0",
		"pkg-dir": "^5.0.0",
		"postcss-custom-properties": "^11.0.0",
```